LICENSE-MIT

```diff
@@ -1,4 +1,5 @@
 Copyright (c) 2015 The trust-dns Developers
+Copyright (c) 2017 Google Inc.
```

Comment:
As a Google employee, Google claims copyright over my patches [1], which is why I am adding Google Inc. as a copyright author for this patch.

[1] https://opensource.google.com/docs/patching/

Reply:
I'm not sure how I feel about this being inserted for the entire project. I'm ok with this being added on a per file basis, but this feels awkward.

Are you making this changes on behalf of Google, on Google time, on Google resources?

Reply:
No, I am not making this change on behalf of Google, on Google time, or on Google resources. However, my employment agreement states that Google owns intellectual property I create while at the company.

I'll ask tomorrow if this patch counts as intellectual property, if it is necessary to update the copyright notices (I think copyright applies nonetheless), and if copyright authors only hold copyright for their changes (and not the entire project).

Reply:
I did not get a clear response about the 2 first points. But adding Google Inc. in LICENSE-MIT does not mean that Google Inc. is a copyright owner for the whole project. It is a copyright owner only for the lines added or modified by a Googler (I still need a definite answer for this too).

I probably won't get updates about this before quite some time (days to months). What I suggest (assuming you prefer to reject this PR which I understand) is that I revert this PR and come back to you when 2.1.0 is out with `BASE32_DNSSEC`. I would then either create an issue to this project for you to create a PR, or create a PR myself if I can point you to a good documentation about the legal consequences of updating the copyright notice.

Does this sound good to you?